### DISCUSSION TITLE
UK streaming content download issue

### DISCUSSION THREAD

**Discussion by TheWallop** [aclap-dev/video-downloadhelper#464] [Help and Q&A]
### Your question

Hi - I need a solution for channel4.com to which I am registered. If I use a trial version of GetFLV, I can download content which gets decrypted without issue. I have two licenses for Video Download Helper - one for Edge, one for Chrome, yet after downloading channel4 content for temporary offline viewing, I only get an encrypted version of the video along with an mp4 file extension. It plays garbled grey. Is there ever going to be a real solution to getting channel4 content using VDH??

PS My license is locked too after a recheck?!!

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-GB
license: locked
key (only 16 first characters): 5af5a2624ae643fb
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Dave",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
hi,
to unlock your license check your spam. If you don't find our email please send us a message at licenses@downloadhelper.net

**Comment by jcfrog** [comment]
Isn't channel4 using DRMs?

**Reply by TheWallop** [reply]
I guess they must be but when GetFLV did it without an issue, I thought... why not VDH??

My need for offline content viewing is legit but I guess I'm being packaged up with all the pirates no doubt.

From a technical perspective, why can GetFLV succeed if it is DRM encrypted?

Thanks

**Reply by jcfrog** [reply]
We don't want to : https://github.com/aclap-dev/video-downloadhelper/wiki/What-can-I-do-if-the-video-is-protected-by-DRM%3F
